robot-core/pom.xml

```diff
@@ -33,50 +33,50 @@
-        <version>0.15.4</version>
-        <configuration>
-          <oldVersion>
-            <dependency>
-              <groupId>${project.groupId}</groupId>
-              <artifactId>${project.artifactId}</artifactId>
-              <version>1.8.3</version> <!-- japicmp target -->
-              <type>jar</type>
-            </dependency>
-          </oldVersion>
-          <newVersion>
-            <file>
-              <path>${project.build.directory}/${project.artifactId}-${project.version}.${project.packaging}</path>
-            </file>
-          </newVersion>
-          <parameter>
-            <!-- see documentation: http://siom79.github.io/japicmp/MavenPlugin.html -->
-            <breakBuildOnBinaryIncompatibleModifications>true</breakBuildOnBinaryIncompatibleModifications>
-            <breakBuildOnSourceIncompatibleModifications>true</breakBuildOnSourceIncompatibleModifications>
-            <!-- include on breaking changes
-            <excludes></excludes>
-            -->
-          </parameter>
-        </configuration>
-        <executions>
-          <execution>
-            <phase>verify</phase>
-            <goals>
-              <goal>cmp</goal>
-            </goals>
-          </execution>
-        </executions>
-      </plugin>
+<!--       <plugin> -->
+<!--         <groupId>com.github.siom79.japicmp</groupId> -->
+<!--         <artifactId>japicmp-maven-plugin</artifactId> -->
+<!--         <version>0.15.4</version> -->
+<!--         <configuration> -->
+<!--           <oldVersion> -->
+<!--             <dependency> -->
+<!--               <groupId>${project.groupId}</groupId> -->
+<!--               <artifactId>${project.artifactId}</artifactId> -->
+<!--               <version>1.8.3</version> japicmp target -->
+<!--               <type>jar</type> -->
+<!--             </dependency> -->
+<!--           </oldVersion> -->
+<!--           <newVersion> -->
+<!--             <file> -->
+<!--               <path>${project.build.directory}/${project.artifactId}-${project.version}.${project.packaging}</path> -->
+<!--             </file> -->
+<!--           </newVersion> -->
+<!--           <parameter> -->
+<!--             see documentation: http://siom79.github.io/japicmp/MavenPlugin.html -->
+<!--             <breakBuildOnBinaryIncompatibleModifications>true</breakBuildOnBinaryIncompatibleModifications> -->
+<!--             <breakBuildOnSourceIncompatibleModifications>true</breakBuildOnSourceIncompatibleModifications> -->
+<!--             include on breaking changes -->
+<!--             <excludes></excludes> -->
+<!--             --> 
+<!--           </parameter> -->
+<!--         </configuration> -->
+<!--         <executions> -->
+<!--           <execution> -->
+<!--             <phase>verify</phase> -->
+<!--             <goals> -->
+<!--               <goal>cmp</goal> -->
+<!--             </goals> -->
+<!--           </execution> -->
+<!--         </executions> -->
+<!--       </plugin> -->
```

Comment:
Can we get some comments on the necessity to comment out the above?

Reply:
Seems I can enable plugin and use this filter:
```Xml
			<ignoreMissingClassesByRegularExpressions>
				     <ignoreMissingClassesByRegularExpression>org.openrdf.rio.*</ignoreMissingClassesByRegularExpression>
			</ignoreMissingClassesByRegularExpressions>
```
Updating the pom accordingly.